tests/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+set(
+    sources
+    0001-multiobj.c
```

Comment:
I sort of agree, but in this case I think I actually prefer the glob approach for the tests.